karman-avionics/src/tasks/README.md

```diff
@@ -1,16 +0,0 @@
-############################
```

Comment:
It looks like you moved this all into the top level readme? I'm fine with that but I go through and organize that a bit or pretty it up a bit with some markdown so if we keep dumping stuff in there it doesn't get difficult to find details.

Reply:
Having README.md only in one place allows it to be the de facto main page. It's one of the settings in the doxygen configuration. Other files should be used for doxygen pages using the /page command.